pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -898,6 +898,28 @@ var _ = Describe("Clone", func() {
 			})
 		})
 
+		Context("Target VM name", func() {
```

Comment:
nit (as the context above says `""generation of target VM""`)
```suggestion
		Context("VM name", func() {
```